### PR TITLE
[General] Improve VDFS article

### PR DESCRIPTION
**Pull request by muczc1wek**
[Demo](https://muczc1wek.github.io/gmc/zengin/general_info/vdfs/)

I don't know if tools section is better in my variant or the previous was more readable

### CODE REVIEW

**Comment by kamilkrzyskow** [comment]
I preferred the dashed list, but perhaps even better would be a table with ❌ and ✔️ for each feature.
CLI: ❌ ✔️ 
Compression: ❌ ✔️

**Comment by muczc1wek** [comment]
obraz

@kamilkrzyskow is that good enough? I have no idea how to style it.

**Comment by auronen** [comment]
The table looks really nice and clear, it will be great once we start adding more vdfs tools.

File: docs/zengin/general_info/vdfs.md [hunk 1]
```diff
@@ -1,24 +1,47 @@
...
+
+Volumes are loaded based on the [timestamp](../tools/vdfs_tools/gothic_vdfs.md#viewer) specified inside the `.vdf` file. The volumes with the latest timestamp are loaded first.
+
+
+### MOD
+`.mod` files are stored in the `Data/ModVDF/` directory and loaded when they are specified in the current game `.ini`.
+```ini title="MyMod.ini"
+// [...]
+[FILES]
+VDF=MyMod.mod  MyMod_Textures.mod
```
Comment (kamilkrzyskow) [inline docs/zengin/general_info/vdfs.md:24]: I think the separator between the `.mod` files matters, as in, it needs to be consistent a `\t` tab or two spaces?
Perhaps it's worth mentioning about it.

**Comment by kamilkrzyskow** [comment]
Apart of the comment LGTM @muczc1wek 👍

The table looks good, if there are more tools we might consider adding a `div` wrapper with some CSS class to center the content, but not necessary currently.

File: docs/zengin/general_info/vdfs.md (hunk 1 above)
Comment (muczc1wek) [reply docs/zengin/general_info/vdfs.md:24]: You are right, two spaces are needed

File: docs/zengin/general_info/vdfs.md [hunk 2]
```diff
@@ -1,24 +1,49 @@
...
+// [...]
+```
+!!! Warning
+    Notice that two spaces are used as a separator between the `.mod` files.
```
Comment (kamilkrzyskow) [inline docs/zengin/general_info/vdfs.md:28]: I'm not sure if this has to be an explicit warning, just a comment inside the MyMod.ini above 🤔 as the .ini file supports comments, but that's a small detail.

**Review by kamilkrzyskow — APPROVED** [review]

**Comment by kamilkrzyskow** [comment]
You can squash merge when you're ready 🫡

File: docs/zengin/general_info/vdfs.md (hunk 2 above)
Comment (muczc1wek) [reply docs/zengin/general_info/vdfs.md:28]: At least it is visible now

**Pull request merged**